lib/fedex/request/tracking_information.rb

```diff
@@ -103,6 +107,21 @@ def invalid_tracking_number?(error_code)
         error_codes = %w{6035 6037 6070 6125 6135 6140 6145 6150 6172 6173 6174 6185 30020 30030 500195 500200 500205}
         error_codes.include? error_code
       end
+
+      def no_tracking_information_available?(error_code)
+        error_codes = %w{6041 9040 9041 9060 500139 500140 500144 500158 500170 500173}
+        error_codes.include? error_code
+      end
+
+      def unable_to_process_request?(error_code)
+        error_codes = %w{3035 3036 3037 3038 3040 3041 3042 3045 3046 3047 3048 3049 3050 3051 3052 3053 3054 3055
```

Comment:
it sucks that we can't just ping an API for these codes - even better would be just one or two codes - great job collecting all these though